src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,70 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it(`should return error for invalid email and valid password`, () => {
+    const invalidEmail = validateRegisterForm('testmail.com', 'P@ssword1!');
+
+    expect(invalidEmail.code).toBe(422);
+    expect(invalidEmail.message).toBe('Email is invalid.');
+  });
+
+  it(`should return error for invalid email and invalid password`, () => {
+    const invalidEmailPassword = validateRegisterForm(
+      'invalid-mail', 'wrong password');
+
+    expect(invalidEmailPassword.code).toBe(500);
```

Comment:
The test case expects a 500 error code for both invalid email and password, which is typically used for server errors. Consider using a 422 error code for validation errors to maintain consistency with other test cases and standard practices.